pkg/ring/lifecycler.go

```diff
@@ -19,15 +19,15 @@ import (
 
 var (
 	consulHeartbeats = promauto.NewCounter(prometheus.CounterOpts{
-		Name: "cortex_ingester_consul_heartbeats_total",
+		Name: "cortex_member_consul_heartbeats_total",
```

Comment:
We should add a label to these metrics so they can differentiate between different rings.